evm/utils.js

```diff
@@ -1017,6 +1022,10 @@ async function getWeightedSigners(config, chain, options) {
     return { signers: [signers], verifierSetId };
 }
 
+const verifyContractByName = (env, chain, contract, args, options = {}, name) => {
```

Comment:
```suggestion
const verifyContractByName = (env, chain, name, contract, args, options = {}) => {
```
options should always be at the end